calendar-manage-events.adoc

```diff
@@ -146,7 +146,7 @@ However, you can revisit the event anytime to mark it as Accept or Decline.
 
 *Decline* image:graphics/close-circle.svg[width=20]: {product-short} deletes the event, and the event does not make it to your calendar.
 
-*Propose a new time* image:graphics/calendar-range.svg[width=20]: {product short} launches the edit event dialog to <<Propose a New Time>> for that event.
```

Comment:
product-short variable is used to update "Zimbra". Not sure changing this to "product short" will update "Zimbra"

Reply:
I couldn't get your point, but as this line will be displayed as:
`Propose a new time calendar range: Zimbra launches the edit event dialog to Propose a New Time for that event.`
(The {product-short} will be replaced by the text "Zimbra")

Reply:
Ok. If {product short} is doing the same as {product-short}, then we should be good.